ui/desktop/app/routes/scopes/scope/projects/settings/index.js

```diff
@@ -13,28 +13,34 @@ export default class ScopesScopeProjectsSettingsIndexRoute extends Route {
     let cacheDaemonStatus,
       formattedCacheVersion,
       formattedCliVersion,
-      formattedDesktopVersion;
+      formattedDesktopVersion,
+      error;
+
     const { versionNumber: cliVersion } =
       await this.ipc.invoke('getCliVersion');
     formattedCliVersion = `v${cliVersion}`;
+
     const { desktopVersion } = await this.ipc.invoke('getDesktopVersion');
     formattedDesktopVersion = `v${desktopVersion}`;
+
     try {
       cacheDaemonStatus = await this.ipc.invoke('cacheDaemonStatus');
     } catch (e) {
-      return {
-        formattedCliVersion,
-        formattedCacheVersion,
-        formattedDesktopVersion,
-        error: e,
-      };
+      error = e;
```

Comment:
non-blocking: Could we change line 28 from `e` to `error`? I know we typically use `e` but in this case I don't see an issue with defining the exception as `error`.